tests/support/bower.json

```diff
@@ -8,6 +8,7 @@
     "purescript-prelude": "1.1.0",
     "purescript-st": "1.0.0-rc.1",
     "purescript-partial": "1.1.2",
-    "purescript-newtype": "0.1.0"
+    "purescript-newtype": "0.1.0",
+    "purescript-generics-rep": "2.0.0"
```

Comment:
I think we definitely should soon, but I'd like to leave it for a separate PR after this one. I think there might be a lot more changes needed than we suspect 😄